xdg-basedir.opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+
+synopsis: "XDG basedir location for data/cache/configuration files"
+
+version: "0.0.4"
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/gildor478/ocaml-xdg-basedir"
+bug-reports: "https://github.com/gildor478/ocaml-xdg-basedir/issues"
+
+authors: [
+  "Sylvain Le Gall"
+]
+maintainer: [
+  "Sylvain Le Gall"
+]
+dev-repo: "https://github.com/gildor478/ocaml-xdg-basedir.git"
+
+depends: [
+  "ocaml"
+  "fileutils"
+  "unix"
+]
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+description: "This library implements the xdg-basedir specification. It helps to define standard locations for configuration, cache and data files in the user directory and on the system. It is a straightforward implementation on UNIX platform and try to apply consistent policies with regard to Windows directories. It is inspired by the Haskell implementation of this specification, and it follows the same choices for Windows directories. [The xdg-basedir specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html) and [the Haskell implementation](http://github.com/willdonnelly/xdg-basedir) and [the API of this implementation](http://xdg-basedir.forge.ocamlcore.org/api)."
```

Comment:
Can you split the description to be multiple lines. A very long line is hard to read.